seed/seed.json

```diff
@@ -29,6 +29,34 @@
                 "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
             }
         },
+        {
+            "name": "CookieListDefaultStudy",
+            "experiments": [
+                {
+                    "name": "Enabled",
+                    "probability_weight": 100,
+                    "feature_assocatiation": {
+                        "enable_feature": ["BraveAdblockCookieListDefault"]
+                    }
+                },
+                {
+                    "name": "Disabled",
+                    "probability_weight": 100,
+                    "feature_assocatiation": {
```

Comment:
this looks funny to me.  This should be 0 right?

Reply:
yes, also I noticed that somehow I managed to copy-paste `feature_assocatiation` :grimacing: 

should be fixed now